org-super-agenda.el

```diff
@@ -647,6 +647,30 @@ available."
           (_ (cl-loop for fn in args
                       thereis (funcall fn item)))))
 
+(org-super-agenda--defgroup property
+  "Group items that contain a property with a given value.
+Argument should be a list with a string representing a
+property, and an optional second argument being either a string
+representing a value, or a function which will be passed the
+found property value, if any. Without a second argument, any item
+with the given property will match. "
+  :section-name (concat "Property: '" (car args) "'"
+                        (if (= 2 (length args)) (concat " = '" (prin1-to-string (nth 1 args)) "'" )))
+  :test  (let* ((prop (car-safe args))
```

Comment:
The `pcase` from the example above could be used to simplify the `:test` form.

Reply:
I think I understood you correctly here, and made the corresponding changes.

Reply:
Thanks.  Could you simplify it a bit more by inlining the `prop` binding?  Since it's only used in `found-value`'s form.

Reply:
Ah, I missed that. Fixed.